csrc/scheduler/transpose.cpp

```diff
@@ -532,6 +570,11 @@ std::string getTransposeRuntimeRejectReason(
   auto innermost_info_entry = getInnerMostDimInfoInReference(
       data_cache, reference_tensors, reference1, domain_map);
   auto innermost_info = innermost_info_entry.get();
+  auto inner_most_pos1_in_ref1 = innermost_info[0];
+  auto inner_most_pos2_in_ref1 = innermost_info[1];
+  if (inner_most_pos1_in_ref1 < 0 || inner_most_pos2_in_ref1 < 0) {
+    return "Transpose scheduler requires exact mapping on inner most dimension on reference tensor.";
```

Comment:
I backed out from the previously manual check on rfactor.

I think this is a more robust way to reject fusion where we can't have an exact mapping of inner_most_dim from other groups on reference tensor.
cc'ing @naoyam